hoomd/mpcd/stream.py

```diff
@@ -743,3 +743,68 @@ def __init__(self, R, density , boundary="no_slip", period=1, seed=234):
                                  bc,
                                  self.density,
                                  self.seed)
+        self._cpp.setFiller(None)
+    def set_filler(self, density, kT, seed, type='A'):
```

Comment:
You don't need to set this to `None`, it will be default initialized as a null pointer.
```suggestion

    def set_filler(self, density, kT, seed, type='A'):
```